p2p/simulations/http.go

```diff
@@ -455,9 +463,21 @@ func (s *Server) StreamNetworkEvents(w http.ResponseWriter, req *http.Request) {
 		}
 	}
 
+	//Only do throttling if requested
+	if req.URL.Query().Get("throttling") == "true" {
+		throttling = true
+		//rate at which we allow events to be emitted on the server-side event stream
+		rate := time.Second / 40
+		throttle = time.Tick(rate)
```

Comment:
I am suggesting to use time.NewTicker and to defer the Stop method to release the resources that the ticker acquires.